bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -333,7 +355,8 @@ class PartialRowsData(object):
                    STATE_ROW_IN_PROGRESS: ROW_IN_PROGRESS,
                    STATE_CELL_IN_PROGRESS: CELL_IN_PROGRESS}
 
-    def __init__(self, read_method, request):
+    def __init__(self, read_method, request,
+                 retry=DEFAULT_RETRY_READ_ROWS):
```

Comment:
The `TestPartialRowsData` testcase needs an assertion in the default `test_constructor`, and another testcase for when the user overrides `retry`.

Reply:
Added assertion in default constructor.

Reply:
@sumit-ql Did you push that change?  I don't see any new code.